src/com/shapestone/fundementals/DataTypes.java

```diff
@@ -0,0 +1,13 @@
+package com.shapestone.fundementals;
+
+public class DataTypes {
+
+	public static void main(String[] args) {
+		
+		byte b = Byte.MAX_VALUE; 
```

Comment:
what are you tring to do here ??

Reply:
printing range of data types. Added  remaining data types